ArduPlane/mode_takeoff.cpp

```diff
@@ -65,7 +65,7 @@ void ModeTakeoff::update()
 {
     if (!takeoff_started) {
         // see if we will skip takeoff as already flying
-        if (plane.is_flying() && plane.ahrs.groundspeed() > 3) {
+        if ( (millis() - plane.started_flying_ms > 10000U) && plane.ahrs.groundspeed() > 3) {
```

Comment:
nitpick: extra space after the bracket. Sorry to be annoying and perhaps not a blocker.

Reply:
fixed